src/components/Button/Button.jsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "./button.scss"
+
+export default function Button(props) {
+    const {className, ...rest} = props;
+
+    return <button className={className ? `btn ${className}` : "btn" } onClick={() => props.onButtonClick()}>{props.title}</button>
+}
```

Comment:
sinse you don't pass any arguments for `props.onButtonClick()` you can write it like this -> `onClick={props.onButtonClick}`  without additional arrow function

Reply:
Thanks, got it